### PR TITLE
Add javadocs

### PR DESCRIPTION
**Pull request by chewwh09**
Add javadocs to Models, Ui and Command Parser.

### CODE REVIEW

File: src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByDecreasingQuantity.java [hunk 1]
```diff
@@ -6,6 +6,12 @@
 
 public class ItemComparatorByDecreasingQuantity implements Comparator<Item> {
 
+    /**
+     * Compares between 2 item by seeing which item has the lower quantity.
```
Comment (Md-Fazil) [inline src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByDecreasingQuantity.java:10]: I think you need a space between the params and description.

File: src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByExpiryDate.java [hunk 2]
```diff
@@ -6,6 +6,12 @@
 
 public class ItemComparatorByExpiryDate implements Comparator<Item> {
 
+    /**
+     * Compares between 2 ExpiryDate objects by seeing which item expires earlier.
```
Comment (Md-Fazil) [inline src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByExpiryDate.java:10]: This also needs a space.

**Review by Md-Fazil — COMMENTED** [review]
Some nits and grits to fix.

File: src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByExpiryDate.java (hunk 2 above)
Comment (chewwh09) [reply src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByExpiryDate.java:10]: noted!

File: src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByDecreasingQuantity.java (hunk 1 above)
Comment (chewwh09) [reply src/main/java/seedu/storemando/model/item/comparator/ItemComparatorByDecreasingQuantity.java:10]: noted!

**Review by Md-Fazil — APPROVED** [review]
LGTM!

**Pull request merged**